apps/prairielearn/vitest.config.ts

```diff
@@ -7,7 +7,19 @@ import { BaseSequencer, type TestSpecification } from 'vitest/node';
 // are slowest. However, this depends on cached data from previous runs, which
 // isn't available in CI. So, we manually specify the slowest tests here and
 // use a custom sequencer to always run them first.
-const SLOW_TESTS = ['src/tests/exampleCourseQuestions.test.ts', 'src/tests/cron.test.ts'];
+const SLOW_TESTS = [
+  // 120s
```

Comment:
Let's not inline specific times here, these will vary wildly based on the machine. For instance, these tests take just 70s on my machine.

Reply:
Removed in [dc60479](https://github.com/PrairieLearn/PrairieLearn/pull/12030/commits/dc60479b2813598c5d7ea3fbbe193a9ef57de49a)